www/install.rst

```diff
@@ -6,13 +6,12 @@ These are instructions for installing :ref:`the full SciPy stack
 <stackspec>`.  For installing individual packages, such as NumPy and
 SciPy, see :ref:`individual-packages` below.
 
-
 Scientific Python distributions
 -------------------------------
 
-For most users, especially on Windows and Mac, the easiest way to install the
-packages of the SciPy stack is to download one of these Python distributions,
-which includes all the key packages:
+For most users, especially on Windows, the easiest way to install the packages
```

Comment:
We should definitely explain the trade-offs. If you have an existing Python install for example, or you don't just do scientific work but also web/desktop/etc development, ``pip`` may be a better fit. The problem is that we don't explain that now anywhere. The result is that way too many new users choosing things like python.org installers, and once they learn more they vote with their feet - towards a distribution (and ``conda``), hardly ever the other way around.